modules/oc-mirror-mirror-to-mirror.adoc

```diff
@@ -40,6 +40,11 @@ $ oc mirror --config=./imageset-config.yaml \// <1>
 . Navigate into the `oc-mirror-workspace/` directory that was generated.
 . Navigate into the results directory, for example, `results-1639608409/`.
 . Verify that YAML files are present for the `ImageContentSourcePolicy` and `CatalogSource` resources.
+
+[NOTE]
+====
+The `repositoryDigestMirrors` section of `ImageContentSourcePolicy` is used for the `install-config.yaml` file during installation.
```

Comment:
Need to clarify what the ImageContentSourcePolicy is here.
```suggestion
The `repositoryDigestMirrors` section of  the`ImageContentSourcePolicy` file/resource is used for the `install-config.yaml` file during installation.
```

Reply:
This sentence was removed by a different writer and no longer appears in the file.

Reply:
Done, thank you!